src/pythie_serving/table_wrapper.py

```diff
@@ -44,25 +44,40 @@ def _create_model_specs(self, model_config: ModelConfig) -> ModelSpecs:
                 value = table_type_mapping[metadata["target_name"]](row[metadata["target_name"]])
                 table[key] = value
 
-        return {
-            "model": table,
-            "feature_names": metadata["feature_names"],
-            "nb_features": len(metadata["feature_names"]),
-            "samples_dtype": object,
-            "extra_specs": None,
-        }
+        extra_specs = None
+        if "default_value" in metadata:
+            default_value = metadata["default_value"]
+            if type(default_value) != table_type_mapping[metadata["target_name"]]:
+                raise PythieServingException(
+                    f"Can't assign default_value {default_value} because it's type {type(default_value)} is "
+                    f"different from target type {table_type_mapping[metadata['target_name']]}"
+                )
+            extra_specs = {"default_value": default_value}
```

Comment:
You could move that to a new method `_parse_extra_specs` to make it easier to read